### DISCUSSION TITLE
Public link to download files

### DISCUSSION THREAD

**Discussion by Avessalom608** [Ylianst/MeshCentral#5875] [Issues] [answered]
As far as I know there should be a public link, why don't I have it?
![image](https://github.com/Ylianst/MeshCentral/assets/40124505/0a44d206-effd-4628-8e95-deca1f3134c3)

**Comment by si458** [comment, marked as answer]
You click the 'tiny 2 blue boxes' next to the file name and it will pop up a box with the url u can share to other people
